workflow {
    throw new Exception("It's gone wrong")
}